klib/ntp.c

```diff
@@ -335,6 +348,16 @@ int init(void *md, klib_get_sym get_sym, klib_add_sym add_sym)
             ntp.pollmin = interval;
         }
     }
+    ntp.reset_threshold = 0;
+    value reset_thresh = get(root, sym_intern(ntp_reset_threshold, intern));
```

Comment:
These next several lines can now be simplified using get_u64.

Reply:
I'm not sure it's worth it in this case--get_u64 doesn't let you make a distinction between a missing value (valid here) vs malformed input (which should be reported as an error). Also, since this is a klib, it's another new symbol to pull in.

Reply:
Yes, that makes sense.